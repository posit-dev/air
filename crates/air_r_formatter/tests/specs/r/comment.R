# Hi there!
1 + 2
