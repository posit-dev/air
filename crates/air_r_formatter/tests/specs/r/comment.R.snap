---
source: crates/air_formatter_test/src/snapshot_builder.rs
info: r/comment.R
---
# Input

```R
# Hi there!
1 + 2

```


=============================

# Outputs

## Output 1

-----
Indent style: Tab
Indent width: 2
Line ending: LF
Line width: 80
-----

```R
# Hi there!
1 + 2
```
